src/webview/uikit/uikit.css

```diff
@@ -334,6 +352,15 @@
   border: 1px solid var(--vscode-button-background);
 }
 
+.info > .button {
+  padding: 2px;
+  font-size: smaller;
+}
+
+.heading.clickable {
+  cursor: pointer;
+}
+
```

Comment:
These two seem way too specific to be in uikit. Let's move them to direct connect styles maybe?